mlir/test/Dialect/Affine/simplify-structures.mlir

```diff
@@ -411,8 +411,6 @@ func.func @test_trivially_false_returning_two_results(%arg0: index) -> (index, i
   // CHECK: %[[c13:.*]] = arith.constant 13 : index
   %c7 = arith.constant 7 : index
   %c13 = arith.constant 13 : index
-  // CHECK: %[[c2:.*]] = arith.constant 2 : index
```

Comment:
Just unnecessary/unrelated to main subject of test?

Reply:
I don't know why the test originally checked for `%c2`. That value is not used. With this commit, the op actually disappears because of the additional notifications, that cause ops to be re-added to the worklist of the greedy pattern rewrite driver. (It is DCE'd away by the greedy pattern rewrite driver.)